sendto_silhouette.py

```diff
@@ -135,6 +138,9 @@ def add_arguments(self, pars):
         pars.add_argument("-D", "--depth",
                 dest = "depth", type = int, default = -1,
                 help="[0..10], or -1 for media default")
+        pars.add_argument("--disable_logging",
+                dest = "disable_logging", type = Boolean, default = False,
+                help="Disable logging to file")
```

Comment:
I'd rather omit yet another parameter but to log always (which won't hurt AFAICT)